pkg/prowgen/prowgen_konflux.go

```diff
@@ -326,6 +327,11 @@ func GenerateKonfluxServerlessOperator(ctx context.Context, openshiftRelease Rep
 			return fmt.Errorf("failed to generate Konflux configurations for %s (%s): %w", r.RepositoryDirectory(), branch, err)
 		}
 
+		csvPath := filepath.Join(r.RepositoryDirectory(), "olm-catalog", "serverless-operator", "manifests", "serverless-operator.clusterserviceversion.yaml")
+		if err := konfluxgen.GenerateReleasePlanAdmission(csvPath, resourceOutputPath, appName, semver.New(soMetadata.Project.Version)); err != nil {
+			return fmt.Errorf("failed to generate ReleasePlanAdmission: %w", err)
+		}
```

Comment:
Can we encapsulate this into the `Generate` function ?

We could accept a CSV path file array config and generate those only when it's passed

Reply:
Not sure, why you mean a file array?
0d680d18